src/python/ttfautohint/__init__.py

```diff
@@ -58,6 +58,8 @@ def __init__(self, path=None, **kwargs):
         version_string = lib.TTF_autohint_version_string().decode('ascii')
         self.version_string = version_string
 
+        lib.TTF_autohint.argtypes = [c_char_p]
```

Comment:
maybe add a comment with a link to our github issue and/or the python bug for future memory.